framework/behaviors/CacheableBehavior.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace yii\behaviors;
+
+use yii\base\Behavior;
+use yii\base\Widget;
+use yii\base\WidgetEvent;
+use yii\caching\Cache;
+use yii\caching\Dependency;
+use yii\di\Instance;
+use yii\web\View;
+
+/**
+ * Cacheable widget behavior.
+ *
+ * @property Widget $owner
+ *
+ * @group behaviors
+ */
+class CacheableBehavior extends Behavior
```

Comment:
This would be inconsistent with the `yii\behaviors\AttributeBehavior`. It's not `yii\behaviors\ActiveRecordAttributeBehavior`.